src/FixedDataTable.js

```diff
@@ -1094,6 +1088,52 @@ class FixedDataTable extends React.Component {
     }
   }
 
+  _didControlledScroll = (/* !object */ nextProps) => {
```

Comment:
Basically, this calls `onScrollStart`, `onVerticalScroll`, `onHorizontalScroll`, and `onScrollEnd` if controlled scrolling has occurred.
Doesn't set the scrolling state, and hence shouldn't trigger a rerender (unless the passed handlers want it that way).